src/main/java/org/jabref/logic/importer/fetcher/transformers/BiodiversityLibraryTransformer.java

```diff
@@ -0,0 +1,97 @@
+package org.jabref.logic.importer.fetcher.transformers;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.jabref.model.strings.StringUtil;
+
+public class BiodiversityLibraryTransformer extends YearRangeByFilteringQueryTransformer {
+
+    private static final List<String> STOP_WORDS = List.of("a", "and", "for", "or", "with");
```

Comment:
I would use `Collection` as datatype (and `HashSet`) as the only use is "contains"?

Reply:
Did you see https://github.com/JabRef/jabref/blob/main/docs/adr/0022-remove-stop-words-during-query-transformation.md? - I think, we remove the stop words.

Reply:
I haven't seen